### PR TITLE
MGMT-19029: Fix issue where validation failed on CNV with ARM host

### PR DESCRIPTION
**Pull request by eifrach**
## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by eifrach** [comment]
test

File: internal/hardware/virt/virt.go [hunk 1]
```diff
@@ -11,5 +12,5 @@ const (
 )
 
 func IsVirtSupported(inventory *models.Inventory) bool {
-	return funk.Contains(inventory.CPU.Flags, intelVirtCpuFlag) || funk.Contains(inventory.CPU.Flags, amdVirtCpuFlag)
+	return slices.Contains(inventory.CPU.Flags, intelVirtCpuFlag) || slices.Contains(inventory.CPU.Flags, amdVirtCpuFlag) || inventory.CPU.Architecture == models.InfraEnvCPUArchitectureArm64
```
Comment (AlonaKaplan) [inline internal/hardware/virt/virt.go:15]: Can you please add a TODO comment telling that we need to find out how to whether ARM cpu host is virt supported?

**Pull request merged**